README.md

```diff
@@ -1 +1,37 @@
-# kris_final_project
\ No newline at end of file
+# kris_final_project # Homes For The Homeless
+
+            ## Description
```

Comment:
Something is wrong with your markdown. If you go to your final project root here. You can see it doesn't look like how it should. I would try to figure out what is causing this. You can all use the markdown preview extension to see how it will look